### ISSUE TITLE
About: List Core Libraries

### ISSUE THREAD

**Issue by pllim** [scientific-python/scientific-python.org#442]
### Please describe the new feature.

In https://github.com/scientific-python/scientific-python.org/blob/main/content/about/_index.md#what-we-do , there is mention of "core libraries" but I cannot immediately find that list. Would be nice if "core libraries" link to that list.

### Add any other context about the new feature here, screenshots, etc.

Scipy 2023 BOF session

**Comment by tupui** [comment]
Thanks, do you want to make a PR @pllim 😃 ?

**Comment by pllim** [comment]
I don't know where the list actually is. 😬

**Comment by stefanv** [comment]
I'm not sure we want to emphasize that idea too much, since the distinction is to some degree arbitrary, but it's here:

https://scientific-python.org/specs/core-projects/

**Comment by pllim** [comment]
From my perspective as "outsider", it is just confusing to see the term "core libraries" without any more info on what those are on that page. So either link to them or remove that term altogether would be less confusing for me. Would be happy to PR if no one get to it first, and you decide what to do. Thanks!

**Comment by jarrodmillman** [comment]
What page?

**Comment by pllim** [comment]
https://scientific-python.org/about/#what-we-do --> "Lecture Notes" bullet point

**Comment by stefanv** [comment]
> We don't want them project maintainers to feel they are being used to promote this project.

I agree with this sentiment, and I think there are very few places outside of the SPECs (where it is structurally helpful) to distinguish between "layers" of projects. Generally, I think it's more of a continuum anyway that stretches from foundational through technique specific through domain specific.

**Comment by jarrodmillman** [comment]
> https://scientific-python.org/about/#what-we-do --> "Lecture Notes" bullet point

Good catch! Yes, let's remove that.

**Comment by pllim** [comment]
Please see https://github.com/scientific-python/scientific-python.org/pull/443
